DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: proofr
 Title: Client for the PROOF API
-Version: 0.3.0.96
+Version: 0.4.0.0
```

Comment:
I'd drop the 4th part of the string - it was just for dev version, so `0.4.0`

Reply:
Good call, updated! Thanks Scott!